tests/conftest.py

```diff
@@ -0,0 +1,10 @@
+import pytest
```

Comment:
Could this go in geomstats/tests.py ? I feel geomstats/tests.py has some configuration wrt tests already, maybe it could be nice to have everything at the same place?